src/main/java/mops/persistence/file/FileTag.java

```diff
@@ -0,0 +1,22 @@
+package mops.persistence.file;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+/**
+ * Short meta information strings.
+ */
+@Data
+@NoArgsConstructor
```

Comment:
Is that allowed with only one field that is `@NonNull`?

Reply:
It needs to be here for Spring Data JDBC.
Per default it uses this empty constructor and the populates the fields with setters or reflection